components/auctions/AuctionsItemPopover.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <Popover>
+    <PopoverButton class="outline-none">
+      <slot />
+    </PopoverButton>
+
+    <transition
+      enter-active-class="transition duration-200 ease-out"
+      enter-from-class="translate-y-1 opacity-0"
+      enter-to-class="translate-y-0 opacity-100"
+      leave-active-class="transition duration-150 ease-in"
+      leave-from-class="translate-y-0 opacity-100"
+      leave-to-class="translate-y-1 opacity-0"
+    >
+      <PopoverPanel
+        class="absolute z-10 rounded-lg border border-slate-600 bg-slate-800 shadow-lg overflow-auto md:-translate-x-1/2 p-3 min-w-72
+        -translate-x-3/4"
+      >
+        <Disclosure v-slot="{ open }">
+          <DisclosureButton class="mt-1">
+            <div class="flex">
+              <UiHeaderLabel
+                :sm="true"
+                class="items-center flex"
+              >
+                <UiIcon
```

Comment:
At least this icon and maybe also the option should be more visible, how about coloring this icon in red?